auth-web/src/views/auth/staff/SetupAccountView.vue

```diff
@@ -2,9 +2,17 @@
   <v-container>
     <div class="view-container">
       <article>
-        <h1 class="mb-3">Create a Director Search Account</h1>
-        <p class="intro-text pb-3">Enter the name and contact information for this account, and indicate which products
-this account will have access to.</p>
+        <div class="d-flex flex-row">
```

Comment:
Remove 'd-flex' and 'flex-row'. Replace with 'view-header' as used in other views.